src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -99,6 +99,28 @@ public void CheckIncludedAssemblies ([Values (false, true)] bool usesAssemblySto
 			}
 		}
 
+		[Test]
+		[Category ("SmokeTests")]
+		[TestCase ("Test Me")]
+		// testing characters as per https://www.compart.com/en/unicode/category/Zs
+		[TestCase ("TestUnicodeSpace0020\u0020Me")]
+		// TODO these break with AOT error on windows
```

Comment:
Is there a way to make the tests run only on macOS and Linux?

Reply:
I can.